src/view/screens/Moderation.tsx

```diff
@@ -69,8 +69,8 @@ export function ModerationScreen({}: Props) {
           style={[styles.linkCard, pal.view]}
           onPress={onPressContentFiltering}
           accessibilityRole="tab"
-          accessibilityHint="Content filtering"
-          accessibilityLabel="">
+          accessibilityLabel={_(msg`Content filtering`)}
+          accessibilityHint="">
```

Comment:
Here, what about "Opens the content filtering preferences"?
```suggestion
          accessibilityHint={_(msg`Opens the content filtering preferences`)}>
```